packages/bpk-component-calendar/src/custom-proptypes.js

```diff
@@ -18,6 +18,22 @@
 
 import PropTypes from 'prop-types';
 
+const CalendarConfigurationSingle = PropTypes.shape({
+  type: 'single',
+  date: PropTypes.instanceOf(Date),
+});
+
+const CalendarConfigurationRange = PropTypes.shape({
+  type: 'range',
+  startDate: PropTypes.instanceOf(Date),
+  endDate: PropTypes.instanceOf(Date),
```

Comment:
Maybe we ought to have a custom prop type that verifies that `endDate > startDate`?